app/_src/networking/transparent-proxy/kubernetes.md

```diff
@@ -0,0 +1,265 @@
+{% endtip %}
+
+<!-- vale Google.Headings = NO -->
+#### ConfigMap lookup strategy
+<!-- vale Google.Headings = YES -->
+
+{{ Kuma }} follows a specific order when searching for the ConfigMap:
+
+- If the workload is annotated with `traffic.kuma.io/transparent-proxy-configmap-name`, {{ Kuma }} first looks for the resource with the specified name in the workload’s namespace. If not found, it will then check the `{{ kuma-system }}` namespace for the same ConfigMap.
+
+- If the workload is not annotated, or the ConfigMap specified in the annotation is not found, {{ Kuma }} will search for the resource named in the `{{ transparentProxy }}.configMap.name` setting. It first looks in the workload's namespace, and if not found, it checks the `{{ kuma-system }}` namespace.
+
+{% warning %}
+The ConfigMap in the `{{ kuma-system }}` namespace is required for proper operation, so it must be present even if custom ones are used in all individual workload namespaces.
+{% endwarning %}
+
+<!-- vale Google.Headings = NO -->
+#### Custom ConfigMap name
+<!-- vale Google.Headings = YES -->
+
+The name of the ConfigMap is defined by the `{{ transparentProxy }}.configMap.name` setting. By default, this name is set to `kuma-transparent-proxy-config`.
+
+To apply a custom name for all workloads, you can modify the `{{ transparentProxy }}.configMap.name` setting during installation:
+
+{% cpinstall transparent-proxy-config-configmap-custom-name-globally %}
+{{ transparentProxy }}.configMap.name=custom-name
+{% endcpinstall %}
+
+To use a different resource for specific workloads, apply the `traffic.kuma.io/transparent-proxy-configmap-name` annotation to those workloads. For example:
+
+```sh
+kubectl annotate pods <pod_name> "traffic.kuma.io/transparent-proxy-configmap-name=custom-name"
+```
+
+This allows flexibility, enabling different workloads to use distinct ConfigMap settings while maintaining a global default.
+
+### Annotations
+
+Kubernetes annotations can be applied to individual workloads to modify the transparent proxy configuration. These annotations allow fine-tuning of specific behaviors for a single workload without affecting others.
+
+{% tip %}
+Before {{ Kuma }} 2.9, annotations were the only way to modify the transparent proxy configuration on a per-workload basis. In {{ Kuma }} 2.9, the [Configuration in ConfigMap](#configuration-in-configmap-experimental) feature was introduced, offering a more efficient way to configure groups of workloads. When this feature is no longer experimental, it will become the recommended approach for managing transparent proxy settings across multiple workloads. Annotations will still be useful for fine-tuning specific configurations when you need an individual workload to behave differently from the others.
+{% endtip %}
+
+Below you can find a list of annotations that can be used to configure specific transparent proxy settings. 
```

Comment:
this list should be moved to reference section

Reply:
addressed